microraiden/microraiden/config.py

```diff
@@ -21,7 +21,11 @@
 
 # address of the default channel manager contract. You can change this using commandline
 # option --channel-manager-address when running the proxy
-CHANNEL_MANAGER_ADDRESS = '0x161a0d7726eb8b86eb587d8bd483be1ce87b0609'  # ropsten
+CHANNEL_MANAGER_ADDRESS = {
```

Comment:
We'll need to store block number in which the contract has been deployed. But this can be done in a separate PR later.
Wouldn't it be better to use simple ints as dict keys instead of string?

Reply:
It's just for convenience, since `web3.version.network` returns strings. Since we're not using anything else for determining the network, I chose this shortcut of directly using strings.